test/Vector.spec.js

```diff
@@ -0,0 +1,41 @@
+QUnit.test("Vector parses notation with a single element", function (assert) {
+    const unit = new Component(1, 1, false, false, false);
+    const inverse = new Component(-1, 1, false, false, false);
+    const expected = [new Vector(unit, unit), new Vector(unit, inverse), new Vector(inverse, unit), new Vector(inverse, inverse)];
+    const actual = Vector.Create("(1, 1)");
+    assert.deepEqual(actual, expected);
+});
+
+QUnit.test("Vector parses notation with a single element and a single semicolon", function (assert) {
+    const unit = new Component(1, 1, false, false, false);
+    const inverse = new Component(-1, 1, false, false, false);
+    const expected = [new Vector(unit, unit), new Vector(unit, inverse), new Vector(inverse, unit), new Vector(inverse, inverse)];
+    const actual = Vector.Create("(1, 1);");
+    assert.deepEqual(actual, expected);
+});
+
+QUnit.test("Vector parses notation with a single element and multiple semicolons", function (assert) {
```

Comment:
Only this test or the one above it is necessary. The part that can cause errors is passing in an empty string, done since only the empty string is after/between semicolon(s). They both test that.